lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1140,10 +1185,128 @@ bool FIRRTLModuleLowering::handleForceNameAnnos(
   return failed;
 }
 
-hw::HWModuleExternOp
+sv::SVVerbatimSourceOp FIRRTLModuleLowering::getVerbatimSourceForExtModule(
+    FExtModuleOp oldModule, Block *topLevelModule,
+    CircuitLoweringState &loweringState) {
+  CircuitNamespace circuitNamespace(loweringState.circuitOp);
+
+  // Check for verbatim black box annotation
+  AnnotationSet annos(oldModule);
+  Annotation verbatimAnno = annos.getAnnotation(verbatimBlackBoxAnnoClass);
+
+  if (!verbatimAnno) {
+    return nullptr;
+  }
```

Comment:
Nit: the braces are unnecessary. Also, later in this member function, the null returns are `return {}`. Use that consistent style throughout (even though it functionally is the same!):
```suggestion
  if (!verbatimAnno)
    return {};
```